wiki/Tournaments/112/Summer_2019/en.md

```diff
@@ -0,0 +1,656 @@
+- DoubleTime
+  - [Hate vs Brilliance – Qubism (Regraz) \[Elexus’ HYPER\]](https://osu.ppy.sh/beatmapsets/524503#osu/1802266)
+
+---
+
+## Match results
+
+### Grand Finals
+
+Saturday, 17 August 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Kwk Yelleri | FF | **1** | **egemen ve aimless** | *-win by default-* |
+
+Monday, 19 August 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| KaFein | 5 | **7** | **egemen ve aimless** | [#1]( https://osu.ppy.sh/community/matches/54199892) |
+
+Thursday, 22 August 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| KaFein | 6 | **7** | **egemen ve aimless** | [#1]( https://osu.ppy.sh/community/matches/54269710) |
+
+### Finals
+
+Thursday, 15 August 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| bum patladın | 3 | **6** | **egemen ve aimless** | [#1]( https://osu.ppy.sh/community/matches/54101219) |
+
+Friday, 16 August 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **KaFein** | **6** | 2 | Kwk Yelleri | [#1]( https://osu.ppy.sh/community/matches/54123533) |
+
+### Semifinals
+
+Sunday, 4 August 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **KaFein** | **5** | 4 | egemen ve aimless | [#1]( https://osu.ppy.sh/community/matches/53834185) |
+| bum patladın | 3 | **5** | **Kwk Yelleri** | [#1]( https://osu.ppy.sh/community/matches/53836911) |
+
+Sunday, 11 August 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **bum patladın** | **6** | 3 | KFC Managers | [#1]( https://osu.ppy.sh/community/matches/54005552) |
+
+Tuesday, 13 August 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **egemen ve aimless** | **6** | 1 | Yürü Baba Fingo | [#1]( https://osu.ppy.sh/community/matches/54055221) |
+
+### Quarterfinals
+
+Saturday, 27 July 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **KaFein** | **5** | 1 | İsim bulamadık pingleme artık | [#1]( https://osu.ppy.sh/community/matches/53645819) |
+
+Sunday, 28 July 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **bum patladın** | **1** | 0 | eTi dasmjasdk | FF |
+| eTi dasmjasdk | 0 | **1** | **Yürü Baba Fingo** | FF |
+| KFC Managers | 3 | **5** | **egemen ve aimless** | [#1]( https://osu.ppy.sh/community/matches/53666015) |
+| **Kwk Yelleri** | **5** | 2 | Yürü Baba Fingo | [#1]( https://osu.ppy.sh/community/matches/53668619) |
+
+Sunday, 4 August 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| İsim bulamadık pingleme artık | 3 | **5** | **KFC Managers** | [#1]( https://osu.ppy.sh/community/matches/53835518) |
+
+### Group Matches
+
+Saturday, 20 July 2019:
+
+| Group | Player 1 | | | Player 2 |
+| :--: | --: | :-: | :-: | :-- |
```

Comment:
```suggestion
| :-: | --: | :-: | :-: | :-- |
```
I found three that has something like this. replace with only one dash for middle alignment